node/runtime/pangolin/src/pallets/bridge_messages.rs

```diff
@@ -16,7 +16,7 @@ frame_support::parameter_types! {
 	// Shared configurations.
 	pub const MaxMessagesToPruneAtOnce: MessageNonce = 8;
 	// TODO: remove this after FeeMarketPayment upgrade
-	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::derive_substrate_address((&b"root"[..]).derive_ethereum_address()));
+	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::derive_substrate_address(H160::from_slice(&b"root0000000000000000"[..])));
```

Comment:
This writing looks not right. @AurevoirXavier

Reply:
> The result is `0x726f6f74303030303030`.

So short? One byte is two hex

Reply:
Yes. Just a part of it.

But I want to show you this is not the right way to pad zero.

Reply:
LGTM.